src/utils/settings/index.js

```diff
@@ -139,6 +148,15 @@ function defaults(custom) {
   // `integrations` returns an array of valid integration items.
   withDefaults.integrations = integrations(withDefaults);
 
+  // validate push options
+  if (typeof withDefaults.streamingEnabled !== 'boolean') withDefaults.streamingEnabled = false;
```

Comment:
Remember you have utilities to use, you should aim to use them as much as possible. Having a single definition of what something is (in this case) allows for easier and faster changes whenever we adapt our rules or want to apply a quick performance optimization.